adaptive_scheduler/scheduler_input.py

```diff
@@ -430,7 +430,7 @@ def _get_available_resources(self):
         resources = []
         for resource_name, resource in self.network_model.items():
             telescope_class = resource_name[:3].lower()  # telescope class is first 3 characters of the resource
-            if not resource['events'] and self.sched_params.telescope_class.lower() in ['all', telescope_class]:
+            if not resource['events']:
```

Comment:
Don't need the `telescope_class` variable anymore I think. Also, is the comment on line 429 still needed?

Reply:
removed